project/app/api/submission.py

```diff
@@ -1,91 +1,252 @@
 import logging
 from fastapi import APIRouter, File, UploadFile
+from fastapi.responses import JSONResponse
 from pydantic import BaseModel, Field, validator
-from app.utils.google_api import GoogleAPI, NoTextFoundException
+from app.utils.img_processing.google_api import GoogleAPI, NoTextFoundException
+from app.utils.complexity.squad_score import squad_score, scaler
+from hashlib import sha512
 import json
+from io import BytesIO
+from requests import get
+import sys
```

Comment:
Could you reorder the imports to be [PEP8 complaint](https://www.python.org/dev/peps/pep-0008/#imports) for ordering? Should be: standard library imports (io, sys), then 3rd party, and then the imports from local files. I had forgotten to be keeping an eye on this piece of PEP8 prior to this PR!

Reply:
Resolved with latest commit.